app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,13 @@
+<footer class="footer">
+  <small>
+    The <a href="http://www.railstutorial.org/"><%= t ".ror_tut" %></a>
```

Comment:
I18n

Reply:
Dạ e tưởng như này là được rồi chị? <%= t ".ror_tut" %>

Reply:
@huongnguyenmta Dạ e tưởng như này là được rồi chị? <%= t ".ror_tut" %>

Reply:
à e nhìn ra rồi :D cái chữ "The" nhỏ quá e không nhìn ra :D